core/autocrypt/account.py

```diff
@@ -295,18 +285,33 @@ def process_incoming(self, msg, delivto=None):
             _, delivto = mime.parse_email_addr(msg.get("Delivered-To"))
             assert delivto
         ident = self.get_identity_from_emailadr([delivto])
+        logger.debug('ident %s', ident)
         if ident is None:
             raise IdentityNotFound("no identity matches emails={}".format([delivto]))
         From = mime.parse_email_addr(msg["From"])[1]
+        logger.debug('From %s', From)
         old = ident.config.peers.get(From, {})
+        logger.debug('old %s', old)
+        logger.debug('msg is still %s', msg)
         d = mime.parse_one_ac_header_from_msg(msg)
+        logger.debug('d %s', d)
         date = msg.get("Date")
         if d and "to" in d:
+            logger.debug('d and to in d')
             if d["to"] == From:
+                logger.debug('to == from')
                 if parsedate(date) >= parsedate(old.get("*date", date)):
+                    logger.debug('date>=old date')
                     d["*date"] = date
                     keydata = b64decode(d["key"])
-                    keyhandle = ident.bingpg.import_keydata(keydata)
+                    logger.debug('type(d[key]) %s', type(d["key"]))
+                    logger.debug('type(keydata) %s', type(keydata))
+                    keyhandle = ident.crypto.import_keydata(keydata)
+                    logger.debug('imported key with keyhandle %s', keyhandle)
+                    pgpykey = ident.crypto.get_publickey_from_kh(keyhandle)
+                    ident.crypto.export_key(pgpykey)
+                    logger.debug('exported key')
+                    ident.crypto.publicpgpykeys.append(pgpykey)
                     d["*keyhandle"] = keyhandle
```

Comment:
please remove most if not all logger.debug() calls -- it's nice during debugging but not as a permanent addition.  Note that you can also do ``import pdb ; pdb.set_trace()`` and use interactive commands to get at variable values so it's not always neccessary to print all values all the time.